File: Questions_in_jS_Console_Based/q17.js
//Accept a day number between 1-7 and print the corresponding day name.

let day = prompt('Enter the Day:. ');
day = day.toLowerCase();
switch (day) {
    case 'sunday':
        alert('Aaj to m GTA 6 khelunga.. pure din...');
        break;
    case 'monday':
        alert('Aaj Puja Krte h Chalo..');
        break;
    case 'tuesday':
        alert('Hamuna Ji ki puja');
        break;
    case 'wednesday':
        alert('Kuch nhi bs Time Kat rha h');
        break;
    case 'thursday':
        alert('Thursday...');
        break;
    case 'friday':
        alert('Friday');
        break;
    case 'saturday':
        alert('Bs Ek aur din aur sunday.. phir.')
        break;

    default:
        break;
}

File: UnDirectional_Folders/Day1_Java/q17.java
import java.util.InputMismatchException;
import java.util.Scanner;

public class q17 {
    public static void main(String[] args) {
        System.out.println("Enter the day between 1 and 7 :");
        try {
            Scanner sc = new Scanner(System.in);
            int day = sc.nextInt();
            switch (day) {
                case 1:
                    System.out.println("Sunday");
                    break;
                case 2:
                    System.out.println("Monday");
                    break;
                case 3:
                    System.out.println("Tuesday");
                    break;
                case 4:
                    System.out.println("Wednesday");
                    break;
                case 5:
                    System.out.println("Thursday");
                    break;
                case 6:
                    System.out.println("Friday");
                    break;
                case 7:
                    System.out.println("Saturday");
                    break;

                default:
                    break;
            }
            sc.close();
        } catch (InputMismatchException e) {
            System.out.println("Enter the Correct Data in Next time.");
        }

    }
}
